generator/src/lib.rs

```diff
@@ -1237,7 +1254,7 @@ pub fn generate_result(ident: Ident, _enum: &vkxml::Enumeration) -> Tokens {
         impl fmt::Display for #ident {
             fn fmt(&self, fmt: &mut fmt::Formatter) -> fmt::Result {
                 let name = match *self {
-                    #(#notation2),*,
+                    #(#display_notation),*,
```

Comment:
`Error::description` is deprecated in favor of `to_string`, so this represents a step backwards in API friendliness to idiomatic code. Users who want the constant's name instead of it's description can already use the `Debug` impl.

Reply:
The debug impl does not represent the constant name, which is why I made the change. The debug impl looks like `Result(1000000)`

Reply:
Many of the constants are not covered by the Debug impls. I'd like to tackle that too, but it's a bigger change and I wanted to get these rolling out.

Reply:
Whoops, looks like I was looking at an old private branch of mine by accident. Regardless, `description` should not be used. For consistency with Rust error enum conventions, the `Debug` impl should display the symbolic name (or a numeric value in newtype struct form if no name is found), and the `Display` impl should provide a written description as it currently already does.

Reply:
Error::description impls are removed